core/include/seqan/align/evaluate_alignment.h

```diff
@@ -252,6 +252,241 @@ TScoreVal computeAlignmentStats(Align<TGaps, TAlignSpec> const & align,
     return computeAlignmentStats(stats, align, scoringScheme);
 }
 
+// ----------------------------------------------------------------------------
+// Function alignmentIdentityString()
+// ----------------------------------------------------------------------------
+
+/*!
+ * @fn Align#alignmentIdentityString()
+ * @headerfile <seqan/align.h>
+ * @brief Returns a CharString of the similarity of positions in an alignment
+ *
+ * @signature CharString alignmentIdentityString();
```

Comment:
To add multiple signatures just add another signature to the @signature command.
Then you would have 2 signatures:
```
@signature alignmentIdentityString(align, ....);
           alignmentIdentityString(gapsH, gapsV, ...);

@param align ....
@param gapsH ....
@param gapsV ....
```